docs/auth/social-auth.md

```diff
@@ -4,6 +4,56 @@ A common misconception is that RNFirebase provides social login out of the box.
 
 Firebase allows a number of social providers to be used out of the box; Facebook, Google, Twitter and Github. You can however choose any provider you wish assuming they have an oAuth API.
 
+## Apple
+
+For Apple Authentication please see our [`@invertase/react-native-apple-authentication`](https://github.com/invertase/react-native-apple-authentication) library which integrates well with Firebase and provides Firebase + Apple Auth examples.
+The [@invertase/react-native-apple-authentication](https://github.com/invertase/react-native-apple-authentication) provides a library that helps manage Apple authentication easily.
+
+**Step 1**: Login with Apple.
+
+```js
+import { firebase } from 'react-native-firebase';
```

Comment:
```suggestion
import firebase from 'react-native-firebase';
```

Not a named export iirc on v5? 🤔

Reply:
Indeed not! Just verified. Wasn't even a typescript-ism sneaking in, thanks for the catch - committing then ready to hit the button again